Source/WebCore/xml/XPathGrammar.h

```diff
@@ -1,26 +1,22 @@
-#ifndef CSSGRAMMAR_H
-#define CSSGRAMMAR_H
-/* A Bison parser, made by GNU Bison 2.3.  */
+/* A Bison parser, made by GNU Bison 3.8.2.  */
 
-/* Skeleton interface for Bison's Yacc-like parsers in C
+/* Bison interface for Yacc-like parsers in C
 
-   Copyright (C) 1984, 1989, 1990, 2000, 2001, 2002, 2003, 2004, 2005, 2006
-   Free Software Foundation, Inc.
+   Copyright (C) 1984, 1989-1990, 2000-2015, 2018-2021 Free Software Foundation,
+   Inc.
 
-   This program is free software; you can redistribute it and/or modify
+   This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
-   the Free Software Foundation; either version 2, or (at your option)
-   any later version.
+   the Free Software Foundation, either version 3 of the License, or
```

Comment:
We can't accept this license change.  That's why we're stuck at an old version.

Reply:
Bison contains a [licensing exception](https://www.gnu.org/software/bison/manual/html_node/Conditions.html) which allows to use its output under any other license, and the exception is itself reproduced in the output files ([implementation](https://github.com/WebKit/WebKit/pull/32672/files#diff-f30aed71e815bddcdab2f2b46e619f5b7980db0ae6d915a64c0885acef25d3a4R25), [header](https://github.com/WebKit/WebKit/pull/32672/files#diff-3006c592ec6e6844ebcc968f77092eb16f1b911051e99bf3ff3c4ed38e91fbc6R21)).

IANAL, but the text is crystal clear in this regard.

Reply:
> We can't accept this license change. That's why we're stuck at an old version.

That would be true if we were shipping Bison itself. We aren't. 

Let me obtain evidence to prove it since I know this is a big deal.

Reply:
/* As a special exception, you may create a larger work that contains
   part or all of the Bison parser skeleton and distribute that work
   under terms of your choice, so long as that work isn't itself a
   parser generator using the skeleton or a modified version thereof
   as a parser skeleton.  Alternatively, if you modify or redistribute
   the parser skeleton itself, you may (at your option) remove this
   special exception, which will cause the skeleton and the resulting
   Bison output files to be licensed under the GNU General Public
   License without this special exception.

   This special exception was added by the Free Software Foundation in
   version 2.2 of Bison.  */